TUM-Dev/NavigaTUM
# Ratelimit refactoring

Fixes #383 

## Proposed Changes (include Screenshots if possible)

- This PR exchanges the custom Ratelimt with [governor](https://crates.io/crates/actix-governor)

## How to test this PR

1. Exchange the `get_token` with the `status` endpont and reduce the ratelimit configuration a bit
    -> See that it correctly Ratelimits 
2. Submit feedback (imo we can test this in production, since I don't see a way this could not work)

## How has this been tested?

- Test that this ratelimits correctly

## Checklist:
